AutoRest/TestServer/swagger/httpInfrastructure.json

```diff
@@ -2795,6 +2795,9 @@
       }
     },
     "A": {
+      "x-ms-exception": {
```

Comment:
Same as ARM. ARM has error named CloudError, but the exception is CloudException.